app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for alpha in phrase:
+        if alpha.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
You could implement this code in one line using chaining and without allocating additional variables. You could think further about this solution later.